tests/mocking/resources/toolbox_test.yaml

```diff
@@ -55,4 +55,17 @@ df_connect:
             jdbc_url: $CONNECT_TOOLBOX_POSTGRES_JDBC
             db_user: "tamr"
             db_password: $CONNECT_TOOLBOX_DB_PASSWORD
-            fetch_size: 10000
\ No newline at end of file
+            fetch_size: 10000
+
+core_connect:
+    host: $TAMR_TOOLBOX_HOST
+    protocol: "http"
+    port: "9050"
+    tamr_username: "admin"
+    tamr_password: $TAMR_TOOLBOX_PASSWORD
+    jdbc:
+        ingest:
+            jdbcUrl: $CONNECT_TOOLBOX_POSTGRES_JDBC
+            dbUsername: "tamr"
+            dbPassword: $CONNECT_TOOLBOX_DB_PASSWORD
+            fetchSize: 10000
```

Comment:
You're missing documentation. But I suppose that can wait until the other comments are addressed.

Reply:
Will add documentation when other comments are addressed.